x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/types.ts

```diff
@@ -39,14 +38,8 @@ export interface EndpointState {
     // Adding `hostInfo` to store full API response in order to support the
     // refactoring effort with AgentStatus component
     hostInfo?: HostInfo;
-    hostDetails: {
-      /** details data for a specific host */
-      details?: Immutable<HostMetadata>;
-      /** details page is retrieving data */
-      detailsLoading: boolean;
-      /** api error from retrieving host details */
-      detailsError?: ServerApiError;
-    };
```

Comment:
This is redundant except for the error and loading. So now we add it on the same level as `hostInfo`

Reply:
Most other changes in a6afaa01b2e6a502241d6079ee4c2fe7dba377d8 is related to this change